sdk2ecr

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+set -e -o pipefail
+shopt -qs failglob
+
+for opt in "$@"; do
+   optarg="$(expr "${opt}" : '[^=]*=\(.*\)')"
+   case "${opt}" in
+      --version=*) VERSION="${optarg}" ;;
+      --registry=*) REGISTRY="${optarg}" ;;
+      --sdk-name=*) SDK_NAME="${optarg}" ;;
+   esac
+done
+
+if [ -z "${VERSION}" ]; then
+  VERSION=$(make print-version | tail -1)
```

Comment:
Alternatively, we can define the version in a **VERSION** file, rather than the Makefile, and remove the `print-version` target.